### PR TITLE
Add Exec path option

### PR DESCRIPTION
**Pull request by Mrono**
Add config option to set exec path of docker-compose exec

### CODE REVIEW

**Review by Steveb-p — APPROVED** [review]
My own personal preference would be naming this as `executablePath`, but I'm ok with this.

It's not like we don't use the abbreviated versions elsewhere :wink: (`cwd`).

File: src/index.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import childProcess from 'child_process'
 import yaml from 'yaml'
 export interface IDockerComposeOptions {
   cwd?: string
+  execPath?: string
```
Comment (AlexZeitler) [inline src/index.ts:5]: I would also prefer `executablePath` as suggested by @Steveb-p.

**Comment by AlexZeitler** [comment]
@Mrono Thanks for your PR. If you can please change the property name as suggested, I'm happy to merge 👍

**Comment by Mrono** [comment]
I've updated the variable name to be more in line with preferred naming rules

**Comment by AlexZeitler** [comment]
@Mrono Thanks again 👍. 📦 0.23.9 is available on npm.

**Pull request merged**